tests/ui/limits/type-length-limit-enforcement.rs

```diff
@@ -1,17 +1,21 @@
+//~ ERROR reached the type-length limit
+
+//! Checks the enforcement of the type-length limit
+//! and its configurability via `#![type_length_limit]`.
+
 //@ build-fail
```

Comment:
Could you try removing this so it becomes the default `//@ check-fail`? I think this should work since it shouldn't have to codegen anything